proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -240,6 +240,10 @@ message WorkflowExecutionInfo {
     // (-- api-linter: core::0142::time-field-type=disabled
     //     aip.dev/not-precedent: This is a vector clock, not a timestamp --)
     int64 task_generation_shard_clock_timestamp = 91;
+
+    VersionedTransition workflow_task_last_update_versioned_transition = 92;
```

Comment:
It will only be updated when the mutation actually counts as a transition. For workflow task it means for transient workflow task updates (e.g. scheduled/start) this field will won't change, and no replication task will be generated. Upon state sync, workflow task state will be sanitized and remove all transient workflow info. 

We can count transient workflow task changes as a transition as well. then we don't need to do the sanitization for transient case (we still need to do it for the speculative case). I think that makes sense and will track the work. Though I will probably do it at the end of the project.